### ISSUE TITLE
train.py: error: unrecognized arguments: --config data_configs/wheat_2020.yaml --project-name fasterrcnn_mobilenetv3_large_fpn_noaug_40e

### ISSUE THREAD

**Issue by WawanFirgiawan** [sovit-123/fasterrcnn-pytorch-training-pipeline#56]
usage: train.py
       [-h]
       [-m MODEL]
       [--data DATA]
       [-d DEVICE]
       [-e EPOCHS]
       [-j WORKERS]
       [-b BATCH]
       [--lr LR]
       [-ims IMGSZ]
       [-n NAME]
       [-vt]
       [-nm]
       [-uta]
       [-ca]
       [-w WEIGHTS]
       [-r]
       [-st]
       [--world-size WORLD_SIZE]
       [--dist-url DIST_URL]
       [-dw]
       [--seed SEED]
train.py: error: unrecognized arguments: --config data_configs/wheat_2020.yaml --project-name fasterrcnn_mobilenetv3_large_fpn_noaug_40e

**Comment by sovit-123** [comment]
@WawanFirgiawan Hello.
I did some argument parser renaming yesterday to make it more similar to the YOLO argument parser as I know many people are familiar with those. Here is the complete list of renaming in case you need a quick lookup:
`--data_configs` => `--data`
`--batch-size` => `--batch`
`--img-size` => `--imgsz`
`--project-name` => `--name`

These are mostly final, and will not be renamed in the future.
